junit.xml

```diff
@@ -1,557 +1,565 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file needed? :)

Reply:
Yep, this file is associated with tests.